src/types/UiaTextRangeBase.cpp

```diff
@@ -1522,6 +1522,14 @@ void UiaTextRangeBase::_moveEndpointByUnitWord(_In_ const int moveCount,
             {
                 success = false;
             }
+            else if (allowBottomExclusive && _tryMoveToWordStart(buffer, documentEnd, resultPos))
+            {
+                // IMPORTANT: _isNextPosWordStart modifies resultPos if successful

```

Comment:
Did you mean `_tryMoveToWordStart`? Otherwise I'm scared about those "is" functions modifying state. 😄

Reply:
yes! Whoops!